specifications/api/swagger.yaml

```diff
@@ -1404,9 +1404,9 @@ x-parametersStakePoolId: &parametersStakePoolId
   required: true
   schema:
     type: string
-    format: hex
+    format: hex|bech32
     maxLength: 56
-    minLength: 64
+    minLength: 56
```

Comment:
Empirically I see that pool ids (no matter if in hex or bech32 format) have consistently 56 char length :man_shrugging:

Reply:
@KtorZ is that something we can guarantee? I'm not sure how deterministic bech32 is here.

Reply:
I don't think this is right indeed, so we could leave it as a comment in the description instead.

Also, pool ids can be 64 chars ..... with jormungandr :s

Reply:
I have added a comment. Pls let me know it that's fine.